goon/code/datums/browserOutput.dm

```diff
@@ -250,14 +250,14 @@ For the main html chat area
 /proc/to_chat(target, message)
 	//Ok so I did my best but I accept that some calls to this will be for shit like sound and images
 	//It stands that we PROBABLY don't want to output those to the browser output so just handle them here
-	if (istype(message, /image) || istype(message, /sound) || istype(target, /savefile) || !(ismob(target) || islist(target) || isclient(target) || istype(target, /datum/log) || target == world))
+	if (IsInteger(target)) //Passing it to an entire z level
```

Comment:
This is just *waaay* too dangerous and error prone. Make it do this when it's passed an actual z level datum only.

Reply:
A zLevel datum is just stats used to construct that z level in /datum/map/proc/addZLevel(), it's not used for storage or logging

Reply:
They definitely are not. See https://github.com/vgstation-coders/vgstation13/blob/bd4f283f571712a51321bdedf5c8f0d3b5c8b754/code/__HELPERS/unsorted.dm#L1327-L1337 and its usages.